draft-ietf-mls-protocol.md

```diff
@@ -2185,6 +2185,11 @@ leaf in the tree, for the second Add, the next empty leaf to the right, etc.
   public key from the KeyPackage in the Add, as well as the credential under
   which the KeyPackage was signed
 
+The identity contained in the credential of the `key_package` should be unique
```

Comment:
should -> SHOULD

Reply:
I thought those capitalized SHOULDs, MUSTs, etc are only used for actual instructions. I use it later on in cases where I describe what the committer SHOULD do. I'm happy to change it here, as well, though.